### PR TITLE
Enable typing!

### PR DESCRIPTION
**Pull request by alexrudd2**
Can I just have a minute of your time to talk about `mypy`?

### CODE REVIEW

**Comment by juli4nb4dillo** [comment]
Testing in my local django.

File: setup.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
     author="Patrick Fuller",
     author_email="pat@numat-tech.com",
     packages=['sartorius'],
+    package_data={'sartorius': ['py.typed']},
```
Comment (juli4nb4dillo) [inline setup.py:17]: I suggest increasing the version so changes are picked up.

Comment (alexrudd2) [reply setup.py:17]: Yeah, I was going to bump for a PyPI release, but can do it now.

**Comment by juli4nb4dillo** [comment]
Tested against 4th floor scale:
image

**Review by juli4nb4dillo — APPROVED** [review]

**Pull request merged**